### PR TITLE
[@kadena/transfer] DetailsCard and Account Text Components

### PR DESCRIPTION
**Pull request by nil-amrutlal-dept**

### CODE REVIEW

File: packages/apps/transfer/src/components/Global/AccountText/index.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+export const AccountText: FC<IAccountTextProps> = ({
+  title,
+  account,
+  defaultVisibility = false,
+}): JSX.Element => {
+  const [visible, setVisible] = useState(defaultVisibility);
+
+  const hiddenAccountValue = `${account.slice(0, 6)}****${account.slice(-4)}`;
+
+  const toogleVisibility = (): void => {
```
Comment (MRVDH) [inline packages/apps/transfer/src/components/Global/AccountText/index.tsx:27]: toggle

File: packages/apps/transfer/src/components/Global/AccountText/styles.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { styled, SystemIcons } from '@kadena/react-components';
+
+export const StyledAccountText = styled('div', {
+  width: '100%',
+});
+
+export const StyledTitle = styled('div', {
+  alignSelf: 'strecth',
```
Comment (MRVDH) [inline packages/apps/transfer/src/components/Global/AccountText/styles.ts:8]: strecth

**Review by MRVDH — APPROVED** [review]
Minor comments.

PS. would it be an idea to also already include the debug-js library here?

**Review by KristinaSpasevska — APPROVED** [review]

File: packages/apps/transfer/src/components/Global/AccountText/styles.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+export const StyledAccountContainer = styled('div', {
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'flex-start',
+  alignSelf: 'stretch',
+  gap: '$2',
+});
+
+export const StyledEyeOffIcon = styled(SystemIcons.EyeOffOutline, {
```
Comment (rezaerami) [inline packages/apps/transfer/src/components/Global/AccountText/styles.ts:32]: please use variables, $6

File: packages/apps/transfer/src/components/Global/DetailsCard/styles.ts [hunk 4]
```diff
@@ -0,0 +1,63 @@
+import { styled } from '@kadena/react-components';
+
+export const StyledInfoItem = styled('div', {
+  display: 'grid',
+  padding: '$3',
+  gridTemplateColumns: 'auto 1fr',
+  gap: '$4',
+  marginTop: '4%',
+  background: 'rgba(71, 79, 82, 0.7)',
+  borderRadius: '4px',
```
Comment (rezaerami) [inline packages/apps/transfer/src/components/Global/DetailsCard/styles.ts:10]: $sm

**Pull request merged**